test/unittests/t_util.cc

```diff
@@ -32,7 +38,94 @@ class ThreadDummy {
 };
 
 
-TEST(T_Util, ThreadProxy) {
+class T_Util : public ::testing::Test {
+ protected:
+  virtual void SetUp() {
+    const bool retval = MkdirDeep(string(sandbox), 0700);
```

Comment:
Funny thing I've realized right now when using CreateTempDir: it doesn't use the second parameter. Temporary directories are always created using by default 0700 permissions.